buildSrc/build.gradle.kts

```
@@ -8,8 +8,8 @@ plugins {
 dependencies {
     implementation(platform(kotlin("bom", "1.7.21")))
```

```suggestion
    implementation(platform(kotlin("bom", "1.7.22")))
```

Why isn't it changed by itself? Consider removing `kotlin(...)` function, it's not Renovate-friendly.
FYI @aSemy

---

Ah, good to know. I haven't set up Renovate myself yet, so I didn't know it was incompatible.

---

See #574

---

Yeah, unfortunately Renovate doesn't actually execute the Gradle scripts, it just parses out the versions using regex.

---

I've cut a feature request: https://github.com/renovatebot/renovate/issues/19147